plugins/tcpudp.yaml

```diff
@@ -0,0 +1,104 @@
+# Plugin Info
+version: 0.0.1
+title: TCP/UDP
+description: TCP/UDP Input Parser
+min_stanza_version: 1.2.7
+parameters:
+  - name: listen_port
+    label: Listen Port
+    description: A port which the agent will listen for syslog messages
+    type: int
+    default: 5140
+  - name: connection_type
+    label: Connection Type
+    description: The type of syslog connection (`udp` or `tcp`)
+    type: enum
+    valid_values:
+      - udp
+      - tcp
+    default: udp
+  - name: log_type
+    label: Type
+    description: Adds the specified 'Type' as a label to each log message.
+    type: string
+    default: tcpudp
```

Comment:
I think this should be conditional based on the connection_type

Reply:
We are splitting this plugin into two separate plugins tcp and udp. I will set default to either tcp or udp respectively.